Extension Project_Final.Rmd

```diff
@@ -0,0 +1,768 @@
+
+##create table with muncode and percent in poverty, in extreme poverty and gini index 
+poor <- coneval %>%
+  filter(!is.na(X3)) %>%
+  select(muncode = X3, pct.poor = X6, 
+         pct.extpoor = X10, gini = X45) 
+
+# Human development index (2010, per capita GNP, schooling, health) ------------
+HDI <- read_csv("data/HDI_2010_UNDP.csv", skip = 3, 
+                col_names = c("state", "mun", "Entidad",  "Municipio",  "school.level",
+                              "expected.school",	"GNP.pc", 	"child.mort",	"education.index",
+                              "income.index", 	"health.index",	"HDI"))
+HDI <- HDI %>%
+  mutate(muncode = state*1000 + mun) %>%
+  select(muncode, HDI)
+
+# police per 1000 pop -----------------------------------------------------
+police <- read_csv("data/police_2010.csv", skip = 2, n_max = 2462, 
+                   col_names = c("muncode", "state", "mun", "total.pers", "pol100k", "pol1k"))
+
+police <- police %>%
+  select(muncode, pol1k) %>%
+  mutate(pol1k = ifelse(pol1k == "n. d.", NA, pol1k))
+```
+
+**Adding Updated Non-Agricultural Variables**
+```{r, warning=FALSE, error=FALSE}
+
+# doctors per 10000 pop ----------------------------------------------------
+doctors <- read_csv("data/doctors_2005_2010.csv", skip = 2, n_max = 2456)
+
+doctors <- doctors %>%
+  select(muncode = Clave, docs = `2010`) %>%
+  mutate(docs_10k = docs * 10) %>% 
+  select(-docs) #making it docs per 10K to correspond with Villarreal
+
+# homicides 2006, 2007, 2008 ----------------------------------------------
+homicides <- read_csv(file="data/homicide_1990_2013_INEGI.csv", col_names = FALSE, skip = 6)
+
+## new coloumn names, default were unreadable
+colnames(homicides) <- c("muncode", "name", 2013:1990)
+homicides[is.na(homicides)] <- 0
+
+##clean out NAs and others, 
+homicides <- homicides %>%
+  filter(!grepl("996|997|998|991|993|992", muncode), muncode > 1000) %>%
+  select(muncode, name, `2008`:`2006`) %>%
+  mutate(hom_total = `2006` + `2007` + `2008`) %>%
+  select(-`2006`, -`2007`, -`2008`, -name)
+
+# area in sqkm ------------------------------------------------------------
+
+map2010 <- read.dbf("data/maps/distritos_2010.dbf")
+map2010 <- tbl_df(map2010)
+
+area <- map2010  %>%
+  mutate(sqkm = (AREA/1000^2)) %>%
+  select(muncode, sqkm)
+
+area.oax <- filter(area, muncode %in% 20001:20030)
+
+# census 2010 -------------------------------------------------------------
+census <- read_delim("data/ITER_NALTXT10.TXT", delim = "\t", col_names = TRUE,
+                     col_types = list(P3YM_HLI = col_numeric(), P8A14AN = col_numeric(), 
+                                      P15YM_AN = col_numeric(), HOGJEF_F = col_numeric(),
+                                      P_15A17_M = col_numeric(), P_18A24_M = col_numeric()))
+
+census[is.na(census)] <- 0
+
+## select relevant census variables
+total_pop <- census %>%
+  select(state = ENTIDAD, mun = MUN, mun.name = NOM_MUN, state.name = NOM_ENT, twn = LOC, pop = POBTOT,
```

Comment:
separate lines for each variable make it easier to read, but that's just my personal preference.